### DISCUSSION TITLE
CDN classes not found in Guides NextJs

### DISCUSSION THREAD

**Discussion by ZeRodolfo** [tailwindlabs/tailwindcss#13352] [Help]
Hello,
I am following this [guide](https://tailwindcss.com/docs/guides/nextjs), but I noticed that some [CDN](https://cdn.tailwindcss.com) classes are not being loaded for this recommended structure for production, for example, these classes `translate-x-full` or `bg-background-paper` are no longer being loaded after the configuration change

**Comment by iazaran** [comment]
No need to see full class name in the minified version.
It can be used like other classes, but this one needs `transform` class too before using `translate-x-full`.
